packages/admin-stories/src/admin/toolbar/samples/TableForm.tsx

```diff
@@ -0,0 +1,152 @@
+import {
+    IStackSwitchApi,
+    MainContent,
+    Stack,
+    StackPage,
+    StackSwitch,
+    Table,
+    Toolbar,
+    ToolbarBackButton,
+    ToolbarTitleItem,
+    useStackSwitchApi,
+} from "@comet/admin";
+import { Add, Delete, Edit } from "@comet/admin-icons";
+import { Button, IconButton, Typography } from "@material-ui/core";
+import * as React from "react";
+
+interface IExampleRow {
+    id: number;
+    foo1: string;
+    foo2: string;
+    nestedFoo: {
+        foo: string;
+    };
+}
+
+const sampleData = [...new Array(40)].map((value, index) => {
+    return { id: index, foo1: `bar-${index}`, foo2: `bar-${index}`, nestedFoo: { foo: `bar-${index}` } };
+});
+
+export const ToolbarTableForm = () => {
+    const [data, setData] = React.useState<Array<IExampleRow>>(sampleData);
+    const onDeleteItem = (id: number) => {
+        setData([...data.filter((item) => item.id != id)]);
+    };
+    const ExampleTable = () => {
+        const stackSwitchApi = useStackSwitchApi();
+        return (
+            <>
+                <Toolbar
+                    actionItems={(stackSwitchApi: IStackSwitchApi | undefined) => (
```

Comment:
Currently, the API for the toolbar is a mixture of `props` and `children`. In my opinion, a more coherent API would be just using `children`, e.g.:

```
<ToolbarActionItems>
...
</ToolbarActionItems>
```

Reply:
Or, we could drop the action items altogether and add a `<ToolbarSpacer/>` component instead, which can be used to move the action item, now a regular `<ToolbarItem />` to the right.

Reply:
I like this direction (more children, less other props with jsx)!

Reply:
i like the solution with  Spacer -> i will call it `<ToolbarFillSpace/>`  and i already implemented it in the last commit